assemblyline/filestore/transport/local.py

```diff
@@ -139,3 +150,16 @@ def _join(base, path):
     if base is None:
         return path
     return os.path.join(base, path.lstrip("/")).replace("\\", "/")
+
+class TransportReadStreamLocal(TransportReadStream):
```

Comment:
You probably don't need this class, and can return the file object directly from read